packages/xo-web/src/xo-app/home/index.js

```diff
@@ -1223,13 +1261,13 @@ export default class Home extends Component {
               ))
             )}
           </div>
-          {filteredItems.length > ITEMS_PER_PAGE && (
+          {filteredItems.length > this._getItemsPerPage() && (
```

Comment:
You can declare a const for it. See https://github.com/vatesfr/xen-orchestra/pull/4708/files#diff-613f8e79185d4e7abf3b39781c0f4d28R1193